### ISSUE TITLE
Unable to start with latest node version on Windows (tested with 18.20.2) - CVE-2024-27980

### ISSUE THREAD

**Issue by klein0r** [node-red/node-red#4653]
### Current Behavior

See https://nodejs.org/en/blog/vulnerability/april-2024-security-releases-2/

> Command injection via args parameter of child_process.spawn without shell option enabled on Windows (CVE-2024-27980) - (HIGH)

https://github.com/nodejs/node/blob/v18.x/doc/changelogs/CHANGELOG_V18.md#18.20.2

Looks like this prevents all commands in `child_process.spawn` ending with .bat or .cmd (on Windows): https://github.com/nodejs/node/commit/6627222409

```
Error: spawn EINVAL
at ChildProcess.spawn (node:internal/child_process:414:11)
at spawn (node:child_process:761:9)
at Object.execFile (node:child_process:351:17)
at I:\ioBroker\node_modules\@node-red\registry\lib\installer.js:581:27
at new Promise ()
at Object.checkPrereq (I:\ioBroker\node_modules\@node-red\registry\lib\installer.js:580:16)
at Object.load (I:\ioBroker\node_modules\@node-red\registry\lib\index.js:59:22)
at I:\ioBroker\node_modules\@node-red\runtime\lib\index.js:162:29
```

And the npm commands end with `.cmd` on Windows:

https://github.com/node-red/node-red/blob/9a3cb0b2b55fc018d7bc450081332b5180a2c7c5/packages/node_modules/%40node-red/registry/lib/installer.js#L28

### Environment

- Node.js version: 18.20.2
- Platform/OS: Windows

**Comment by knolleary** [comment]
The fix is in progress - https://github.com/node-red/node-red/pull/4652

**Comment by knolleary** [comment]
Fixed in Node-RED 3.1.9
